apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -901,7 +901,13 @@ protected int update(int nextUpdate) {
             case 352:
                 apply("ALTER TABLE transaction ALTER COLUMN signature VARBINARY NULL DEFAULT NULL");
             case 353:
-                return 353;
+                apply("ALTER TABLE scan ADD current_height INT NOT NULL DEFAULT 0");
```

Comment:
Don't forget to move that SQL into new FlyWay scripts on develop.

Reply:
Already done, pr is on the way